feature-libs/organization/administration/components/user-group/users/user-group-user-list.component.ts

```diff
@@ -45,7 +66,7 @@ export class UserGroupUserListComponent {
   }
 
   protected notify(item: UserGroup) {
-    this.messageService.add({
+    this.subList.messageService.add({
```

Comment:
Breaking change here? We could check if `messageService` is added to the constructor (deprecated version) and if it isn't, use `subList.messageService` (latest version) so that we do not need to import `featureLevelService` to check if the version is `3.2`.

```
if (this.messageService) {
	this.messageService.add( ...
} else {
	this.subList.messageService.add( ...
}

```

Reply:
O dont think so my change requires that. I dont use new dependency, viewchild with new property will work without any customer action
@marlass please confirm

Reply:
I actually agree with Caine, but would go the different way.

Have if to check if you have `subList` defined, as if the component would be extended by customer and the template would not be updated then it might fails. And the additional fallback then if messageService exists would be good.